1017-login-page-design-guidance.md

```diff
@@ -0,0 +1,20 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-22
+---
+
+# Login Page Design Guidance
+
+The [login page](https://clarity.design/documentation/login) is a predefined form for applications that require authentication.
+
+## Guidance
+
+- **Must** use login page component for authentication flows to maintain consistency across the applications.
+- **Must** follow the [password design guidance](/1019) for the password input.
+- The username/password validation message **must not** call out whether the error occurred in the username or password field.
```

Comment:
```suggestion
- Design teams **must not** specify whether the error occurred in the username or password field in the validation message, as this can compromise security and lead to unauthorized access attempts.
```